runtime/module.go

```diff
@@ -50,7 +51,9 @@ func ProvideCodecs(moduleBasics map[string]AppModuleBasicWrapper) (
 
 	// build codecs
 	basicManager := module.BasicManager{}
-	for name, wrapper := range moduleBasics {
+	names := maps.Keys(moduleBasics)
+	for _, name := range names {
+		wrapper := moduleBasics[name]
```

Comment:
This code isn't sorting the keys, so AFAICT doesn't produce behavior any different than ranging over the map directly?